site_scons/scons2rst.py

```diff
@@ -0,0 +1,350 @@
+#!/usr/bin/env python3
+# encoding: utf-8
+
+# This file is part of Cantera. See License.txt in the top-level directory or
+# at https://cantera.org/license.txt for license and copyright information.
+
+"""
+scons2rst.py: Extract SCons configuration options from SConstruct
+"""
```

Comment:
IMO, this would be better living the in website as a plugin for Nikola, so that it could run each time the website is generated and we don't have to remember to download/extract/upload the zip file from the artifacts. I'm happy to help turn this into a plugin over there.

Reply:
I thought of that, but I believe that wouldn't cover changes in `SConstruct` on the main repo. The converter script relies on some conventions (e.g. it relies on variable names and the convention that the first assignment of a new class variable to `defaults` contains critical information). This PR would cover the conversion as part of CI, where a failing test would automatically point towards the issue, which isn't possible if this would be moved to `cantera-website`.